filebeat/module/nginx/access/ingest/default.json

```diff
@@ -4,7 +4,7 @@
     "grok": {
       "field": "message",
       "patterns":[
-        "\"?%{IP_LIST:nginx.access.remote_ip_list} - %{DATA:nginx.access.user_name} \\[%{HTTPDATE:nginx.access.time}\\] \"%{GREEDYDATA:nginx.access.info}\" %{NUMBER:nginx.access.response_code} %{NUMBER:nginx.access.body_sent.bytes} \"%{DATA:nginx.access.referrer}\" \"%{DATA:nginx.access.agent}\""
+        "\"?%{IP_LIST:network.forwarded_ip} - %{DATA:user.name} \\[%{HTTPDATE:nginx.access.time}\\] \"%{GREEDYDATA:nginx.access.info}\" %{NUMBER:http.response.status_code} %{NUMBER:nginx.access.body_sent.bytes} \"%{DATA:http.request.referrer}\" \"%{DATA:nginx.access.agent}\""
```

Comment:
Is the first ip in this list also `source.ip`?

Reply:
I always forget in which order the forwarded IPs are. But yes, either first or last is the actual source IP, and the rest of the array is the proxies along the way.

Reply:
Just double checked, and in our expected log, the remote IP is the first-ish in the array.

Looking carefully, I realize that the whole purpose of the painless script in there is to determine the remote IP by looking for the first public IP in the array. Barring that, it falls back to item `[0]`. Perhaps proxies populate this inconsistently, not sure why that's necessary otherwise...